ajax/libs/jmespath/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "jmespath",
+  "version": "0.15.0",
+  "filename": "jmespath.min.js",
+  "description": "JMESPath implementation in JavaScript",
+  "homepage": "http://jmespath.org/",
+  "keywords": [
+    "jmespath",
+    "jsonpath",
```

Comment:
This keyword will direct users to touch another library called JsonPath. I think it's better to remove this.